akka-stream/src/main/boilerplate/akka/stream/javadsl/GraphCreate.scala.template

```diff
@@ -40,4 +43,24 @@ private[stream] abstract class GraphCreate {
     scaladsl.GraphDSL.create([#g1#])(combineMat.apply) { b => ([#s1#]) => block.apply(b.asJava, [#s1#]) }#
   
   ]
+
+  /**
+     * Creates a new [[Graph]] by importing the given graph list `graphs` and passing their [[Shape]]s
+     * along with the [[GraphDSL.Builder]] to the given create function.
+     */
+
+    def create[IS <: Shape, S <: Shape, M, G <: Graph[IS, M]](graphs: java.util.List[G],
+        block: function.Function2[GraphDSL.Builder[Seq[M]], java.util.List[IS], S]): Graph[S, java.util.List[M]] = {
```

Comment:
Type parameter to `GraphDSL.Builder` is still a Scala `Seq` here, should be `j.u.List`